test/helpers.js

```diff
@@ -0,0 +1,154 @@
+import debugModule from "debug";
+const debug = debugModule("test:helpers");
+
+import path from "path";
+import fs from "fs-extra";
+import async from "async";
+import Contracts from "truffle-core/lib/contracts";
+import Debug from "truffle-core/lib/debug";
```

Comment:
**Open concern**: these two imports are causing problems with CI.

CI checks are [failing](https://travis-ci.org/trufflesuite/truffle-debugger/jobs/332105182#L903) because of what appears to be a problem with circular dependencies during checkout. In this case:
- **truffle-debugger** requires **truffle-core** as a `devDependency`.
- **truffle-core** requires truffle-debugger as a regular `dependency`.

It seems there are a few options but I am wondering if people are opposed to the creation of two additional **truffle-`*`-utils** repos: one for **truffle-core**'s compilation/artifact-writing (**truffle-compile-utils**), and one for the debug CLI message formatting (**truffle-debug-utils**).